gradle-plugin/build.gradle

```diff
@@ -76,6 +76,10 @@ kotlin {
     explicitApi()
 }
 
+tasks.withType(ValidatePlugins).configureEach {
+    it.enableStricterValidation = true
```

Comment:
I didn't know about this.

Reply:
why is it not the default lol

Reply:
Chime in here please, https://github.com/gradle/gradle/issues/22600